apps/els_lsp/src/els_diagnostics_utils.erl

```diff
@@ -25,6 +26,26 @@ included_uris(Document) ->
   POIs = els_dt_document:pois(Document, [include, include_lib]),
   included_uris([Id || #{id := Id} <- POIs], []).
 
+-spec included_documents(els_dt_document:item()) -> [els_dt_document:item()].
```

Comment:
An alternative could be to reuse the `included_uris/1` function here and to `filtermap` on them with the `els_utils:lookup_document/1`.

Reply:
Did in fcab36c.